### PR TITLE
feat(manager): Workflow 3.0 

### PR DESCRIPTION
**Pull request by nikhitamore**
## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+//               >
+//                 {alert.props.description}
+//               </div>
+//             </div>
+//           ))}
+//         </div>
+//       )}
+//     </div>
+//   );
+// };
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx:178]: Why the code is commented

File: packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionContent,
+  AccordionToggle
+} from '@patternfly/react-core';
+import { useEffect, useState } from 'react';
+
+const data1 = {
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx:188]:
```suggestion
const data = {
```

File: packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+      _id: '643fee8c48fdf0c31b012fd6',
+      propertyIdentifier: 'testnikki',
+      action: 'FORM_UPLOADED',
+      props: {
+        applicationIdentifier: 'dev',
+        env: 'dev'
+      },
+      message: 'Deployment Time : 11.09 seconds [Containerized]',
+      payload:
+        '{"accessUrl":"https://testnikki-dev.apps.int.mpp.preprod.iad2.dc.paas.redhat.com/"}',
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx:200]: Don't expose this urls

File: packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+      _id: '643fa45f48fdf0c31b012740',
+      propertyIdentifier: 'testnikki',
+      action: 'BUILD_STARTED',
+      props: {
+        applicationIdentifier: 'dev',
+        env: 'dev'
+      },
+      message: 'Deployment Time : 12.53 seconds [Containerized]',
+      payload:
+        '{"accessUrl":"https://testnikki-dev.apps.int.mpp.preprod.iad2.dc.paas.redhat.com/"}',
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx:234]: Same here

File: packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+//         ...state,
+//         step: currentStep
+//       });
+//       console.log(state.step);
+//     }
+//   }, 3000);
+//   return () => clearInterval(interval);
+// }, []);
+
+// const currentStepObj = steps.find((step) => step.step === state.step);
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/Notification.tsx:359]:
```suggestion
// useEffect(() => {
//   let currentStep = state.step;
//   const interval = setInterval(() => {
//     currentStep++;
//     if (currentStep > 5) {
//       clearInterval(interval);
//     } else {
//       setState({
//         ...state,
//         step: currentStep
//       });
//       console.log(state.step);
//     }
//   }, 3000);
//   return () => clearInterval(interval);
// }, []);

// const currentStepObj = steps.find((step) => step.step === state.step);
```

Why this code is commented

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx [hunk 6]
```diff
@@ -34,55 +35,58 @@ export const SideBar = () => {
...
-        </Nav>
+        <>
+          <Nav aria-label="Nav">
+            <NavList>
+              <Link href={pageLinks.webPropertyListPage}>
+                <a className="text-decoration-none">
+                  <SidebarNavItem
+                    title="Web Properties"
+                    icon={<ThIcon size="sm" />}
+                    isActive={pathname === pageLinks.webPropertyListPage}
```
Comment (riginoommen) [inline packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx:46]:
```suggestion
                    isActive={pathName === pageLinks.webPropertyListPage}
```

File: packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx [hunk 7]
```diff
@@ -487,14 +503,24 @@ export const WebPropertyDetailPage = (): JSX.Element => {
...
+        style={{ minHeight: '600px' }}
       >
-        <SSRForm
+        <AddDeplyoment
           propertyIdentifier={propertyIdentifier}
           onClose={() => handlePopUpClose('createSSRDeployment')}
         />
+
+        {/* <SSRForm
+          propertyIdentifier={propertyIdentifier}
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx:517]: Why here also commented code

File: packages/manager/src/views/WebPropertyDetailPage/components/DeploymentProgress/DeploymentProgress.tsx [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+    message: 'ENV_CREATION access provided for souchowd@redhat.com',
+    payload:
+      '{"name":"Soumyadip Chowdhury","email":"souchowd@redhat.com","propertyIdentifier":"testnikki","action":"ENV_CREATION","createdBy":"nmore@redhat.com","updatedBy":"nmore@redhat.com"}',
+    source: 'GIT',
+    createdBy: 'nmore@redhat.com',
+    createdAt: '2023-04-12T13:04:24.986Z',
+    updatedAt: '2023-04-12T13:04:24.986Z',
+    __v: 0
+  },
+  // {
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/DeploymentProgress/DeploymentProgress.tsx:106]: Please remove this commented code

File: packages/manager/src/views/WebPropertyDetailPage/components/DeploymentProgress/DeploymentProgress.tsx [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+    .filter((item) => item.source === 'GIT')
+    .map((item) => ({ action: getStepVariant(item.action), des: item.action }));
+
+  const [stepStatuses, setStepStatuses] = useState(data.map(() => 'info'));
+
+  useEffect(() => {
+    if (currentStep >= data.length) return;
+
+    const newData = [...stepStatuses];
+    // newData[currentStep] = ProgressVariant.success;
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/DeploymentProgress/DeploymentProgress.tsx:274]:
```suggestion

```

File: packages/manager/src/views/WebPropertyDetailPage/components/addDeployment.tsx [hunk 10]
```diff
@@ -0,0 +1,122 @@
+// import { Switch, Tooltip } from '@patternfly/react-core';
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/addDeployment.tsx:1]: Why this code block is commented

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Comment by nikhitamore** [comment]
Changing branch

**Pull request closed without merging**